apps/services/user-profile/src/app/user-profile/dto/userDeviceTokens.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { IsDate, IsString } from 'class-validator'
+import { IsNationalId } from '@island.is/nest/validators'
+import { Type } from 'class-transformer'
+
+const EXAMPLE_TOKEN =
+  'f4XghAZSRs6L-RNWRo9-Mw:APA91bFGgAc-0rhMgeHCDvkMJBH_nU4dApG6qqATliEbPs9xXf5n7EJ7FiAjJ6NNCHMBKdqHMdLrkaFHxuShzTwmZquyCjchuVMwAGmlwdXY8vZWnVqvMVItYn5lfIH-mR7Q9FvnNlhv'
+
+export class UserDeviceTokensDto {
+  @ApiProperty({
+    required: true,
+    example: 'b3f99e48-57e6-4d30-a933-1304dad40c62',
+  })
+  @IsString()
+  id!: string
+
+  @ApiProperty({ required: true, example: '1305775399' })
```

Comment:
nationalId of real person? if so consider using `0101307789` (Gervimaður útlönd)